models/models.go

```diff
@@ -138,10 +138,13 @@ func getEngine() (*xorm.Engine, error) {
 		return nil, err
 	}
 
-	engine, err := xorm.NewEngine(setting.Database.Type, connStr)
+	engine, err := xorm.NewEngineWithParams(setting.Database.Type, connStr)
```

Comment:
`xorm.NewEngineWithParams` needs third params.

Reply:
Damn, did not undo enough changes :) This change was not needed